src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/impl/extension/BitbucketEnvVarExtension/buildEnv.groovy

```diff
@@ -0,0 +1,10 @@
+package com.cloudbees.jenkins.plugins.bitbucket.impl.extension.BitbucketEnvVarExtension
+
+def l = namespace(lib.JenkinsTagLib)
+
+// Base on javadoc in EnvironmentContributor Jenkins provides other extension points (such as SCM) to contribute environment variables to builds, and for those plugins, Jenkins also looks for /buildEnv.groovy and aggregates them.
```

Comment:
Does this file actually work? I cannot think how it would.

Reply:
Indeed it does not. Serms buildEnv.groovy is ignored for GitSCMExtension

Reply:
I have create this https://issues.jenkins.io/browse/JENKINS-75072